src/styles/partials/components/_alert.scss

```diff
@@ -0,0 +1,115 @@
+.dg_alert {
+  background: $vibrant-blue;
+  padding: $default-padding;
+
+  .dg_alert__container {
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+
```

Comment:
Remove extra space on line 9.

Reply:
space replaced with justify-content: center for default alerts